### PR TITLE
[fixes #103407558] added Request403Error

### PR DESCRIPTION
**Pull request by alexdesi**
fixes  #103407558

### CODE REVIEW

File: lib/hammer.rb [hunk 1]
```diff
@@ -17,9 +17,11 @@ module Errors
     autoload :ValidationError, 'hammer/errors/validation_error'
     autoload :RequestError,    'hammer/errors/request_error'
     autoload :Request400Error, 'hammer/errors/request400_error'
+    autoload :Request403Error, 'hammer/errors/request403_error'
     autoload :Request404Error, 'hammer/errors/request404_error'
     autoload :Request422Error, 'hammer/errors/request422_error'
     autoload :ResponseError,   'hammer/errors/response_error'
+    autoload :ForbiddenError,  'hammer/errors/forbidden_error'
```
Comment (alaibe) [inline lib/hammer.rb]: move on spanner

Comment (os6sense) [inline lib/hammer.rb]: Should ResponseError & ForbiddenError contain the HTTP response code  (i.e. Forbidden403Error) or vice-versa for consistency?

Comment (alexdesi) [inline lib/hammer.rb]: @os6sense I agree, actually I am moving ForbiddenError on spanner (as @alaibe wrote), and there it's not important the code error.

**Comment by alaibe** [comment]
:+1:

**Comment by os6sense** [comment]
:+1:

**Pull request merged**